### PR TITLE
update .gitignore to ignore .toml

### PR DESCRIPTION
**Pull request by bitkarrot**
Prevent accidental .toml checkins for fly.io deploys!

### CODE REVIEW

**Comment by dni** [comment]
i am not sure about this ignore, maybe only add the fly.toml only, because what about ```pyproject.toml``` you could probably exclude that also? imho this first is better.

File: .gitignore [hunk 1]
```diff
@@ -38,3 +38,6 @@ docker
 
 # Nix
 *result*
+
+# fly.io
+*.toml
```
Comment (callebtc) [inline .gitignore:43]:
```suggestion
fly.toml
```

Comment (callebtc) [reply .gitignore:43]: You ok with this @bitkarrot?

Comment (bitkarrot) [reply .gitignore:43]: yup @callebtc

File: .gitignore [hunk 2]
```diff
@@ -40,6 +40,9 @@ docker
 # Nix
 *result*
 
+# fly.io
+fly.toml
+
 # Ignore extensions (post installable extension PR)
 extensions/
```
Comment (callebtc) [inline .gitignore:47]:
```suggestion
```

**Pull request merged**